src/new-client/vite.config.ts

```diff
@@ -124,6 +124,12 @@ const setConfig = ({ mode }) => {
     },
     server: {
       port: 1917,
+      proxy: !process.env.VITE_MOCKS && {
+        "/ws": {
+          target: "http://localhost:55000",
```

Comment:
Where does this url come from exactly? port 55000 sounds oddly specific, was this something you mapped to locally to connect to the locally running container of the gateway?

Reply:
Docker - just updating this now so the default (if no VITE_MOCKS) will route through to our one and only Hydra endpoint (demo)